tests/langs/mod.rs

```diff
@@ -73,763 +66,764 @@ impl InScopeLinePart {
-    C::new(CodeQuery::Prepared(PreparedCQuery::Enum)),
+    c::CompiledQuery::new(&c::PreparedQuery::Enum.into()),
 )]
 #[case(
     "base.c_struct",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Struct)),
+    c::CompiledQuery::new(&c::PreparedQuery::Struct.into()),
 )]
 #[case(
     "base.c_variable",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Variable)),
+    c::CompiledQuery::new(&c::PreparedQuery::Variable.into()),
 )]
 #[case(
     "base.c_function",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Function)),
+    c::CompiledQuery::new(&c::PreparedQuery::Function.into()),
 )]
 #[case(
     "base.c_function_definition",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::FunctionDef)),
+    c::CompiledQuery::new(&c::PreparedQuery::FunctionDef.into()),
 )]
 #[case(
     "base.c_function_declaration",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::FunctionDecl)),
+    c::CompiledQuery::new(&c::PreparedQuery::FunctionDecl.into()),
 )]
 #[case(
     "base.c_switch",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Switch)),
+    c::CompiledQuery::new(&c::PreparedQuery::Switch.into()),
 )]
 #[case(
     "base.c_if",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::If)),
+    c::CompiledQuery::new(&c::PreparedQuery::If.into()),
 )]
 #[case(
     "base.c_for",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::For)),
+    c::CompiledQuery::new(&c::PreparedQuery::For.into()),
 )]
 #[case(
     "base.c_while",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::While)),
+    c::CompiledQuery::new(&c::PreparedQuery::While.into()),
 )]
 #[case(
     "base.c_do",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Do)),
+    c::CompiledQuery::new(&c::PreparedQuery::Do.into()),
 )]
 #[case(
     "base.c_union",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Union)),
+    c::CompiledQuery::new(&c::PreparedQuery::Union.into()),
 )]
 #[case(
     "base.c_identifier",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Identifier)),
+    c::CompiledQuery::new(&c::PreparedQuery::Identifier.into()),
 )]
 #[case(
     "base.c_declaration",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::Declaration)),
+    c::CompiledQuery::new(&c::PreparedQuery::Declaration.into()),
 )]
 #[case(
     "base.c_callexpr",
     include_str!("c/base.c"),
-    C::new(CodeQuery::Prepared(PreparedCQuery::CallExpression)),
+    c::CompiledQuery::new(&c::PreparedQuery::CallExpression.into()),
 )]
 fn test_language_scopers(
     #[case] snapshot_name: &str,
     #[case] contents: &str,
-    #[case] lang: impl LanguageScoper,
+    #[case] lang: Result<impl LanguageScoper, tree_sitter::QueryError>,
```

Comment:
nit: `tree_sitter::QueryError` -> `langs::TSQueryError`, aka the reexported one from `langs/mod.rs`?

Reply:
Fixed in https://github.com/alexpovel/srgn/pull/144/commits/5c08ce08de67f30c77d691e4c0309efbe31f2bec